.travis.yml

```diff
@@ -1,18 +1,20 @@
 language: java
+
 jdk:
     - openjdk7
     - oraclejdk7
     - oraclejdk8
+
 addons:
-    postgresql: "9.3"
+    postgresql: "9.5"
```

Comment:
9.5 is not available yet in travis addons. If you really want to use 9.5 there, you should additionally install packages:
```yaml
addons:
  apt:
    packages:
    - postgresql-contrib-9.5  # will cover most of the things you need
  postgresql: "9.5"
```

But it looks like there are no integration-tests executed, because build did not failed.